### PR TITLE
Make "connect" button blue and change the copy a bit

### LINKED ISSUE #389: Make the "connect" button of variant "primary"
The current color of the "Connect" button lacks sufficient contrast against the background colour and is not unique in comparison to other components in the interface.  Establishing a connection to the dApp is a pre-requisite for most feature interactions other than watch mode. 

The current low contrast can make it difficult for it to stand out as a prominent part of the interface, especially for those with visual impairments. 

If we make it easier for all users to visually distinguish we may increase the attention to this button and increase the chance of users connecting their wallet as one of their first interactions with Multix. 

273064579-8dce769e-146d-429b-b291-551f582fc5ff

Acceptance Criteria: The new colour should provide a clear and distinct visual separation from the background and other components, drawing attention and making it easier for all users to identify.

### PR DESCRIPTION
**Pull request by Tbaut**
closes #389 

There's 1 case when we have 2 buttons primary on a page, but it's doing the same thing, so I guess it's ok. 

- Connect and Connect wallet buttons are now blue
- Change the copy "Watch One" -> "Wach address" since the "one" we're referring to makes no sense, one what, wallet?
- "Connect extension" to "Connect wallet" to be future proof if we want to use WalletConnect or anything else

I used this opportunity to refactor the Home that was having a lot of duplicate code. This means please check that the correct messages  are displayed when they should (BTW let's make a test for this, edit: https://github.com/ChainSafe/Multix/issues/412)
- Multix welcome message when brand new landing
- No account found when clicking on connect without an extension
- No multisig found when an account is shared in the extension but there's no msig and watched address is empty

image
image
image

Submission checklist:

#### Layout

- [x] Change inspected in the desktop web ui
- [x] Change inspected in the mobile web ui

#### Compatibility

- [x] Functionality of change validated with a connected account with multisig
- [x] Applicable elements hidden / disabled for watched multisigs / pure

### CODE REVIEW

**Review by Lykhoyda — APPROVED** [review]
Looks good 👍

File: packages/ui/src/components/ConnectCreateOrWatch.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  const navigate = useNavigate()
+
+  return (
+    <WrapperStyled>
+      {isAllowedToConnectToExtension ? (
+        <div>No multisig found for your accounts or watched accounts.</div>
+      ) : (
+        <>
+          <h1>Multix is an interface to easily manage complex multisigs.</h1>
+          <div>Connect a wallet to interact with Multix or watch an address.</div>
```
Comment (asnaith) [inline packages/ui/src/components/ConnectCreateOrWatch.tsx:18]:
```suggestion
          <div>Connect a wallet to interact with Multix or watch an account.</div>
```

We should update this too now the button name has changed

**Comment by asnaith** [comment]
Left one little suggestion for consistency but other than that looks great. 

The appropriate message is displayed for all of the scenarios listed.

File: packages/ui/src/components/ConnectCreateOrWatch.tsx (hunk 1 above)
Comment (Tbaut) [reply packages/ui/src/components/ConnectCreateOrWatch.tsx]: Makes sense, thanks!

**Review by asnaith — APPROVED** [review]

**Pull request merged**